python/cudf_polars/cudf_polars/dsl/expressions/boolean.py

```diff
@@ -85,6 +86,13 @@ def __init__(
             BooleanFunction.Name.IsLastDistinct,
             BooleanFunction.Name.IsUnique,
         )
+        if self.name in {
+            BooleanFunction.Name.IsBetween,
```

Comment:
Only `IsClose` is new, if I'm reading this correctly. Do we have a test case for `IsBetween`?